### ISSUE TITLE
Investigate how to pull appellant POA

### ISSUE THREAD

**Issue by laurjpeterson** [department-of-veterans-affairs/caseflow#10431]
When the appellant is not the Veteran, we still need to show the appellant's POA. We've gotten one [support ticket](https://github.com/department-of-veterans-affairs/appeals-support/issues/4977) where VBMS is showing the appellant's POA, but Caseflow show there is no POA. 

## AC
- If the appellant is not the Veteran, populate appellant POA in the POA section of Case Details.
- This applies to both Legacy and AMA cases

## Other notes
- VSOs need to see cases where the appellant is not the Veteran, and they need to see their appellant's cases. So, we probably need to enhance the `TrackVeteranTask` to populate for appellants, too. 🤔

**Comment by msj0nes** [comment]
# Support Team Update 
This issue still impacting users and needs to be prioritized to resolved reported issues.

# YourIT Tickets Impacted 
 
YourIT Ticket  Number | Date Ticket Created | Case Details URL 
-- | -- | -- 
INC8867687 | 12/25/19 |   https://appeals.cf.ds.va.gov/queue/appeals/4008482, https://appeals.cf.ds.va.gov/queue/appeals/556a20f4-9ec1-4178-86b5-ea74f8bad416

**Comment by hschallhorn** [comment]
[Slack chatta](https://dsva.slack.com/archives/CHX8FMP28/p1579612996036300)

**Comment by hschallhorn** [comment]
```ruby
org = Vso.find(8)
=> #<Vso id: 8, type: "Vso", name: "Disabled American Veterans", ... >
user = org.admins.first
RequestStore[:current_user] = user

uuids = ["42dde0a6-384b-495f-b1bc-8ccb72df4b10", "5e437b69-0143-42c5-bdf2-5d2f229f3f46", "09535e74-c53a-4ea6-bf75-aece6191f4e6", "fb6e4f0c-4f4d-40cc-9dfe-2b3ce2e03c63", "1af32f64-2814-4bce-aa6b-53d1fc8bb27d", "52b5c3a1-ff35-4403-a08e-e80285aac0c2", "e3cee50b-4617-465b-8ff4-8c9069f0bda6", "93e63346-a403-45a7-81bb-87059d57f72f", "03b8767c-c715-4fee-a169-5968231466f4", "83a304a3-31ff-4e79-8454-07945206ac35", "f982abf5-aaab-4635-a6ae-31b8504711b5", "22e8fca2-8b2e-4b09-8208-48e62b8594cb", "783bc090-4fff-4f1c-88b6-336e8c2c7bfd", "280db2f9-d970-4c83-9345-595c9bdf3c03", "8910db33-49d3-44c5-a47f-9937347f87ac"]
appeals = Appeal.where(uuid: uuids)

appeals.map(&:veteran_is_not_claimant)
=> [true, true, true, true, true, true, true, true, false, false, false, true, true, true, true]
appeals.map(&:representative_name).uniq
=> ["DISABLED AMERICAN VETERANS"]
appeals.map { |a| BGSService.new.can_access?(a.veteran_file_number) }
=> [false, false, false, false, false, false, false, false, false, false, false, false, false, false, false]
```
[Logs](https://console.amazonaws-us-gov.com/cloudwatch/home?region=us-gov-west-1#logEventViewer:group=dsva-appeals-certification-prod/opt/caseflow-certification/src/log/caseflow-certification.out;stream=i-0bc55680454551ff9;filter=%223a4d0a55-a209-41d5-9e9b-a6c19c4052cb%22;start=2019-12-12T16:49:00Z) for BVARSPICER attempting to access appeal 1af32f64-2814-4bce-aa6b-53d1fc8bb27d.
```ruby
user = User.find_by(css_id: "BVARSPICER")
RequestStore[:current_user] = user
user.organizations.first.name
=> "Disabled American Veterans"
appeal = Appeal.find_by(uuid: "1af32f64-2814-4bce-aa6b-53d1fc8bb27d")
user.participant_id
=> "886428"
org.participant_id
=> "2452340"
appeal.claimant.representative_participant_id
=> "2452340"
BGSService.new.can_access?(appeal.veteran_file_number)
=> false
```

Looks like all we need to do is check to make sure the user is part of the representative org of the claimant if `can_access?` fails

**Comment by pkarman** [comment]
I believe this is implemented correctly now for LegacyAppeal as of ec6565b97a1d81fa49afb7fef5c64549af8630ef and may also be working correctly for AMA appeals.

We are implementing in EE via https://github.com/department-of-veterans-affairs/caseflow-efolder/issues/1218

I suggest we re-test all the known instances of this issue are still relevant.

**Comment by pkarman** [comment]
Relevant BGS calls used now in EE include https://github.com/department-of-veterans-affairs/ruby-bgs/blob/master/lib/bgs/services/claimant.rb#L23
